src/sentry/static/sentry/app/views/organizationGroupDetails/groupTagValues.tsx

```diff
@@ -118,7 +118,7 @@ class GroupTagValues extends AsyncComponent<
           </a>
           <DataExport
             payload={{
-              queryType: 2,
+              queryType: 'Issues-by-Tag',
```

Comment:
This is where the 'string vs. int' is implemented in practice. I find it much more readable, and simple to understand what the context of the query is, without having to find the python dict connecting the two.